packages/cli/src/commands/create/__tests__/index.test.ts

```diff
@@ -0,0 +1,57 @@
+import { vi } from 'vitest'
+import { create } from '../'
+import {
+  isValidPackageManager,
+  selectPackageManager,
+  selectRepositoryStructure,
+  isValidStructure,
+} from '../../../utils'
+
+vi.mock('../../../utils')
```

Comment:
I might have too little knowledge on this one, but is there a reason why this file imports utils in three different ways? 

See line 8,10 and 16

Reply:
Ah, let me explain.

By running `vi.mock('../../../utils')`, we mock everything that is exported from `utils`.

Among those mocked exports, we're importing these 4 functions:

```js
import {
  isValidPackageManager,
  selectPackageManager,
  selectRepositoryStructure,
  isValidStructure,
} from '../../../utils'
```

The mocked functions do nothing, but we are importing these functions to assign a mock implementation for testing purpose. For example, we can do things like:

```js
vi.mocked(isValidPackageManager).mockImplementation(() => true);

vi.mocked(isValidPackageManager).mockImplementation(() => false);

vi.mocked(isValidPackageManager).mockImplementationOnce(() => true);
```

Then... what about line 19?

```js
utils = await vi.importActual('../../../utils')
```

This actually imports the unmocked real version of utils module.
Although we mocked everything, we sometimes need the real implementation of a certain methods.

For example, in this test case, we're using all the mocked things from `utils` except for this `isValidPackageManager` method. I'm putting the actual implementation back.

https://github.com/storyblok/field-plugin/blob/2ba6f82528166dd0f2b7422627d2a9d011087036/packages/cli/src/commands/create/__tests__/index.test.ts#L26C1-L33

You can read more about this at https://vitest.dev/guide/mocking.html

Let me know if you have any question :)